webapp/packages/plugin-help/src/Shortcuts/Shortcut.tsx

```diff
@@ -24,12 +30,19 @@ export const Shortcut: React.FC<Props> = function Shortcut({ shortcut }) {
     <div className={s(style, { shortcutContainer: true })}>
       <div className={s(style, { shortcutLabel: true })}>{translate(shortcut.label)}</div>
       <div className={s(style, { shortcutContent: true })}>
-        {shortcut.code.map((code, index) => (
-          <React.Fragment key={code}>
-            {index > 0 && <span className={s(style, { span: true })}>{translate('ui_or')}</span>}
-            <div className={s(style, { shortcutCode: true })}>{code}</div>
-          </React.Fragment>
-        ))}
+        {shortcut.code.map((code, index) => {
+          const formattedCode = code
+            .split(' ')
+            .map(code => CODE_FORMAT_MAP[code.toLowerCase()] ?? code)
+            .join(' ');
```

Comment:
Probably it should be handled in the `transformKeys`
[transformKeys](https://github.com/dbeaver/cloudbeaver/blob/0edfb29851a80780a11d959eac043beb3794dc85/webapp/packages/plugin-help/src/Shortcuts/SHORTCUTS_DATA.ts#L102)